sklearn/cluster/k_means_.py

```diff
@@ -803,12 +804,12 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
     ...               [4, 2], [4, 4], [4, 0]])
     >>> kmeans = KMeans(n_clusters=2, random_state=0).fit(X)
     >>> kmeans.labels_
-    array([0, 0, 0, 1, 1, 1], dtype=int32)
+    array([1, 0, 1, 0, 0, 1], dtype=int32)
```

Comment:
This looks a bit fishy ... You would expect that the first three points (x=1) cluster together and that the last three (x=4) cluster together ...

![figure](https://user-images.githubusercontent.com/1680079/30731446-ac271468-9f6d-11e7-9b1b-c1efacc4194f.png)

Reply:
It looks like this particular problem is sensitive to the value of `random_state`. This docstring example  with `random_state=0` (shown below)

![image](https://user-images.githubusercontent.com/11656932/30761515-24c9784e-9fa4-11e7-9a79-38b1ddea5849.png)

gives a different clustering than, for example, `random_state=1`.

![image](https://user-images.githubusercontent.com/11656932/30761541-39256af0-9fa4-11e7-90ff-f4ee9691e000.png)